lib/view_model/wallet_list/wallet_list_view_model.dart

```diff
@@ -70,6 +70,8 @@ abstract class WalletListViewModelBase with Store {
 
   @action
   Future<void> loadWallet(WalletListItem walletItem) async {
+    bool switchingToSameWalletType = walletItem.type == _appStore.wallet?.type;
+    await _appStore.wallet?.close(shouldCleanup: !switchingToSameWalletType);
```

Comment:
🤔 so I was attempting to handle both cases of
- switching between wallets
- restoring/creating a new wallet
this code handles the first case, but not the second

closing the wallet twice really shouldn't have any side effects, but I agree it could be cleaner

Reply:
actually since we aren't cleaning up mwebd anymore this /should/ be safe to remove